native/package.json

```diff
@@ -29,8 +29,7 @@
     "flow:install-types": "flow-typed install --ignoreDeps dev",
     "flow:install-types:all": "flow-typed install",
     "postinstall": "jetifier",
-    "locales:missing": "node locales/tools/findMissingLocales",
-    "circleci:update-config": "cd .circleci && { cat autogenerated_header.yml; circleci config pack src; } > config.yml"
+    "locales:missing": "node locales/tools/findMissingLocales"
```

Comment:
can be removed, as max will move the locales to the root

Reply:
Then max will do that :D